src/goto-symex/renaming_level.h

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************\
+
+Module: Symbolic Execution
+
+Author: Romain Brenguier, romain.brenguier@diffblue.com
+
+\*******************************************************************/
+
+/// \file
+/// Renaming levels
+
+#ifndef CPROVER_GOTO_SYMEX_RENAMING_LEVEL_H
+#define CPROVER_GOTO_SYMEX_RENAMING_LEVEL_H
+
+#include <map>
+#include <unordered_set>
+
+#include <util/irep.h>
+#include <util/ssa_expr.h>
+
+struct renaming_levelt
```

Comment:
Would be good to add also some documentation at this occasion.

Reply:
I added some documentation now.